test/cpp_extensions/open_registration_extension.cpp

```diff
@@ -23,13 +33,41 @@ C10_REGISTER_GUARD_IMPL(PrivateUse1, c10::impl::NoOpDeviceGuardImpl<DeviceType::
 
 }} // namespace at::detail
 
+namespace {
+
+void abs_kernel(::at::TensorIteratorBase& iter) {
+  // Since this custom device is just for testing, not bothering to implement kernels.
+  abs_counter += 1;
+}
+
+} // namespace
+
+namespace at::native {
+
+REGISTER_PRIVATEUSE1_DISPATCH(abs_stub, &abs_kernel);
+
+} // namespace at::native
+
 // basic dummy add function
 at::Tensor custom_add_Tensor(const at::Tensor & self, const at::Tensor & other, const at::Scalar & alpha) {
   add_counter += 1;
   // Since this custom device is just for testing, not bothering to implement kernels.
   return at::empty(self.sizes(), self.options());
 }
 
+// basic abs function
+at::Tensor& custom_abs_out(const at::Tensor& self, at::Tensor& out) {
+  return at::native::abs_out(self, out);
+}
+
+const at::Tensor& custom_resize_(
+    const at::Tensor& self,
+    c10::IntArrayRef size,
+    c10::optional<c10::MemoryFormat> optional_memory_format) {
+  // Since this custom device is just for testing, not bothering to implement kernels.
+  return self;
```

Comment:
it looks like it's failing because `custom_resize_()` is already defined further down.

You can just use the existing one instead of defining this new one.

Reply:
ok, I have updated the code

Reply:
all checks have passed, could this PR be merged?